packages/firestore/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
   `FirestoreSettings` to `true`. Note that the current behavior
   (`DocumentSnapshot`s returning JS Date objects) will be removed in a future
   release. `Timestamp` supports higher precision than JS Date.
+- [feature] Added ability to control whether DocumentReference.get() and
+  Query.get() should fetch from server only, cache only, or attempt server and
```

Comment:
To make this maximally helpful (especially since there's generally no statement completion in JS and docs will probably not be updated immediately), I'd probably say:

... should fetch from server only (by passing `{ source: 'server' }`, cache only (by passing `{ source: 'cache' }`), or attempt ...

Reply:
Done.